float64 velocity_error
float64 omega_error
float64 gamma_error
---